proto/mls/message_contents/group_mutable_metadata.proto

```diff
@@ -14,9 +14,9 @@ message GroupMutableMetadataV1 {
   // Creator starts as only super_admin
   // Only super_admin can add/remove other super_admin
   Inboxes super_admin_list = 3;
-  // The ED25519 private key used to sign commit log entries
-  // Must match the first entry in the commit log to be valid
-  optional bytes commit_log_signer = 4;
```

Comment:
This is not related to oneshot messages - removed as this is not being used in any live client